doc/classes/SpatialMaterial.xml

```diff
@@ -42,6 +45,7 @@
 			<argument index="1" name="enable" type="bool">
 			</argument>
 			<description>
+				If [code]true[/code], enables the specified [enum Feature]. Many features that are available in SpatialMaterials need to be enabled before use. This way the cost for using the feature is only incurred when specified. Features can also be enabled by setting the corresponding member to [code]true[/code].
```

Comment:
Remember that you can use `[SpatialMaterial]s` without fear here, `makerst.py` will convert it to "`SpatialMaterial`s" without redundant hyperlink.

Reply:
Great to know! I was worried about creating a recursive monster